jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -372,6 +376,84 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         managerState.flags[StateFlags.IS_ONGOING] = false
     }
 
+    /**
+    helper function to generate 1p dependency metrics
+    makes GET requests to maven central to check for 1p dependencies for a small subset of the project's
+    dependencies
+     */
+    private fun makeGETRequests(groupId: String, artifactId: String, version: String, jobId: JobId): Boolean {
+        try{
+            val response = getTextFromUrl("https://search.maven.org/solrsearch/select?q=g:$groupId%20AND%20a:$artifactId%20AND%20v:$version%20AND%20p:jar&rows=20&wt=json")
+                try {
+                    val json: JsonObject = Json.parseToJsonElement(response) as JsonObject;
+                    val responses = json["response"] as? JsonObject;
+                    val numFound = responses?.get("numFound");
+
+
+                    if(numFound.toString() == "0"){
+                        CodetransformTelemetry.projectContains1pDependencies(
+                            codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
+                            codeTransformJobId = jobId.id,
+                            has1pdependency = true
+                        )
+                        return true;
+                    }
+                } catch (e: Exception){
+                    LOG.warn{"Error parsing JSON response from GET request to maven central: ${e.printStackTrace()}"}
```

Comment:
don't print stack traces directly

```suggestion
                    LOG.warn(e) {"Error parsing JSON response from GET request to maven central"}
```